### DISCUSSION TITLE
ESP32C6的AT固件在哪下载？

### DISCUSSION THREAD

**Discussion by 006hjy** [espressif/esp-at#757] [Q&A] [answered]
ESP32C6的AT固件在哪下载？现在八月都过了？？？

**Comment by ustccw** [comment]
请参考 https://docs.espressif.com/projects/esp-at/zh_CN/latest/esp32c6/Compile_and_Develop/How_to_download_the_latest_temporary_version_of_AT_from_github.html 下载。

**Comment by ustccw** [comment, marked as answer]
您可以从这里下载 ESP32-C6 AT 固件：https://docs.espressif.com/projects/esp-at/zh_CN/latest/esp32c6/AT_Binary_Lists/esp_at_binaries.html#esp32-c6-4mb
